src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -443,4 +443,8 @@
 {2} - severity, for example low, moderate, high, or critical
 {3} - URL for more info on the known vulnerability</comment>
   </data>
+  <data name="Error_HttpSource_Single" xml:space="preserve">
+    <value>You are running the '{0}' operation with an 'HTTP' source: {1}. NuGet requires HTTPS sources. To use an HTTP source, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere.</value>
```

Comment:
FYI, I'm sure you know this, but wanted to point out this aka.ms link currently results in a 404.

Reply:
Yes, the aka.ms will be fixed with the docs